src/main/resources/static/js/admin/report-management.js

```
@@ -0,0 +1,287 @@
+            }
+        } catch (error) {
+            console.error(`${type} 카운트 업데이트 실패:`, error);
+        }
+    }
+
+    // 각 타입별 대기중 개수를 정확히 가져오는 메서드들
+    async getPopupPendingCount() {
+        const params = new URLSearchParams({
+            targetType: 'POPUP',
+            status: 'OPEN',
+            page: '0',
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)